PROJECT

```diff
@@ -11,7 +12,7 @@ resources:
     namespaced: true
   controller: true
   domain: cockroachlabs.com
-  group: crdb.cockroachlabs.com
```

Comment:
Is this correct?

Reply:
Yes, the domain and group are joined together during code-gen.

Reply:
Verified with a new project to see what is there. We can see this in the demo memcached-operator from the operator framework:

Project file: <https://github.com/operator-framework/operator-sdk/blob/master/testdata/go/v3/memcached-operator/PROJECT>
GV Info: <https://github.com/operator-framework/operator-sdk/blob/master/testdata/go/v3/memcached-operator/api/v1alpha1/groupversion_info.go#L29>

The group definition should be comprised of the group + domain in the PROJECT file